docs/usage/cluster_scoped_argocd.md

```diff
@@ -0,0 +1,86 @@
+# Cluster Configuration
+
+When the user wishes to install Argo CD with the purpose of managing cluster resources, Argo CD is granted permissions to manage specific cluster-scoped resources which include platform operators, optional OLM operators, user management, etc. 
+
+Note: **Argo CD is not granted cluster-admin**.
+
+## Argo CD Installation
+
+To manage cluster-config, deploy an ArgoCD instance using the steps provided above. 
+
+* As an admin, update the existing Subscription Object for Argo CD Operator and add `ARGOCD_CLUSTER_CONFIG_NAMESPACES` to the spec.
+
+Edit the Subscription yaml to add ENV, **ARGOCD_CLUSTER_CONFIG_NAMESPACES** as defined below.
+
+```
+spec:
+  config:
+    env:
+    - name: ARGOCD_CLUSTER_CONFIG_NAMESPACES
+      value: <namespace where Argo CD instance is installed>, <another namespace where Argo CD instance is installed>
+```
+
+###  In-built permissions for cluster configuration
+
+<table>
+  <tr>
+    <td>Resource Groups</td>
+    <td>What does it configure for the user/administrator</td>
+  </tr>
+  <tr>
+    <td>storage.k8s.io</td>
+    <td>Storage.</td>
+  </tr>
+  <tr>
+    <td>operators.coreos.com</td>
+    <td>Optional operators managed by OLM</td>
+  </tr>
+  <tr>
+    <td>user.openshift.io , rbac.authorization.k8s.io</td>
+    <td>Groups, Users and their permissions.</td>
+  </tr>
+  <tr>
+    <td>config.openshift.io</td>
+    <td>Control plane Operators managed by CVO used to configure cluster-wide build configuration, registry configuration, scheduler policies, etc.</td>
```

Comment:
argocd-operator doesn't explicitly set permissions for OpenShift cluster config resources. Looking at the code the operator is setting the below policy rules for cluster roles:

```
apiGroups: '*'
resources: '*'
verbs: '*'
```

Isn't it the same privileges of a cluster admin? :thinking: